inference-engine/tests/functional/inference_engine/ngraph_reader/ti.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include <string>
 #include "ngraph_reader_tests.hpp"
-TEST_F(NGraphReaderTests, ReadTensorIteratorNetwork) {
+TEST_F(NGraphReaderTests, DISABLED_ReadTensorIteratorNetwork) {
```

Comment:
Is it ok that the tests are disabled?

Reply:
these changes were moved to a separate branch